docs/getting-started/advantages.md

```diff
@@ -0,0 +1,24 @@
+---
+template: main.html
+---
+
+# Advantages
+First, Iter8 has no restrictions on the types of resources that make up a version of an application. This includes custom resources; that is, those defined by a custom resource definition (CRD). Because the set of resources that comprise a version is declarative, it is easy to [extend](../user-guide/topics/extensions.md). Note that this same extension mechanism also allows Iter8 to be used with any service mesh.
```

Comment:
> it is easy to [extend](../user-guide/topics/extensions.md)

What is it? Be specific

Reply:
```suggestion
First, Iter8 has no restrictions on the types of resources that make up a version of an application. This includes custom resources; that is, those defined by a custom resource definition (CRD). Because the set of resources that comprise a version is declarative, it is easy to [extend](../user-guide/topics/extensions.md). This extension mechanism also allows Iter8 to be used with any service mesh.
```

Reply:
done